code/style.css

```diff
@@ -1,7 +1,278 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0px;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #75cfb8;
+} 
 
-.active + .answer {
-  /* some styling here */
+/*------Header starts here------ */
+.header {
+  position: relative;
 }
+  
+.hero-image {
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+}
+  
+.hero-picture {
+  position: absolute;
+  top: 80px;
+  right: 0;
+  bottom: 0;
+  left: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+}
+
+/*------Header ends here------*/
+
+/*------Navbar starts here------*/
+
+.container {
+  position: absolute;
+  top: 0; 
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: auto;
+}
+
+/* The navbar */
+.topnav {
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+}
+
+/* Navbar links */
+.topnav a {
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 20px 22px; 
+  text-decoration: none;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.topnav a:hover {
+  background-color: #ffc478;
+  opacity: 0.70;
+  color: black;
+  border-radius: 25%;
+}
+
+/*------Navbar ends here------*/
+
+.flex-parent {
+  height: 1200px;
+  margin: 5px;
+}
+    
+.flex-child {
+  width: 100%;
+  height: 100%;
+  border-radius: 10%;
+}
+    
+.one{
+  background:#bbdfc8;
+  text-align: center;
+}
+    
+.two {
+  background: #bbdfc8;
+  margin-top: 5px;
+  text-align: center;
+}
+
+h1 {
+  font-size: 32px;
+}
+
+  
+/*------Accordion section starts here------*/
+
+.accordion {
+  border-radius: 7px;
+}
+      
+.accordion-header {
+  margin: 0;
+  padding: 10px;
+  font-size: 28px;
+}
+      
+.accordion-content {
+  display: none;
+  margin: 0;
+  padding: 10px;
+  border-top: 3px solid #ffc478;
+}
+      
+.accordion-selected { /* The content section thats pops up when clicking to the question */
+  display: block;
+}
+      
+.active, .accordion-content, .accordion:hover { /* Color of the table after clicking */
+  background-color: #f0e5d8; 
+  cursor: pointer;
+}
+      
+.accordion-header:after { /* Plus sign, as default */
+  content: "\002B"; /* Plus sign */
+  color: #ffc478;
+  font-weight: bold;
+  float: right;
+  margin-left: 5px;
+}
+      
+.active:after { /* Minus sign after clicking to the question */
+content: "\2212"; /* Minus sign */
+} 
+
+/*------Accordion section ends here------*/
+
+/*------Form section starts here------*/
+      
+form p {
+  margin-bottom: 5px;
+  font-size: 28px;
+}
+
+form input {
+  font-size: 28px;
+}
+
+input:valid {
+  background-color: #ffc478;
+}
+      
+input:invalid {
+  background-color: #75cfb8;
+}
```

Comment:
Nice!

Reply:
Would be nice with a pattern attribute for the telephone number in the HTML form :)